homeassistant/components/backup/util.py

```diff
@@ -8,10 +8,17 @@
 
 from homeassistant.util.json import JsonObjectType, json_loads_object
 
-from .const import BUF_SIZE
+from .const import BUF_SIZE, LOGGER
 from .models import AddonInfo, AgentBackup, Folder
 
 
+def make_backup_dir(path: Path) -> None:
+    """Create a backup directory if it does not exist."""
+    if not path.exists():
```

Comment:
Maybe use `exist_ok` in `path.mkdir` instead?

https://docs.python.org/3/library/pathlib.html#pathlib.Path.mkdir